server/src/test/java/org/opensearch/action/admin/indices/close/TransportVerifyShardBeforeCloseActionTests.java

```diff
@@ -333,16 +335,47 @@ public void testGetReplicationModeWithRemoteTranslog() {
         TransportVerifyShardBeforeCloseAction action = createAction();
         final IndexShard indexShard = mock(IndexShard.class);
         when(indexShard.isRemoteTranslogEnabled()).thenReturn(true);
+        when(indexShard.indexSettings()).thenReturn(createIndexSettings(true));
+        when(indexShard.routingEntry()).thenReturn(createShardRouting(true, true));
```

Comment:
If we change from `indexShard.routingEntry().isAssignedToRemoteStoreNode()` to `indexShard.isRemoteNode()`, we would need to change this as well.